sig/transaction/README.md

```diff
@@ -0,0 +1,28 @@
+# Transaction Special Interest Group
+
+Covers transaction related work in TiKV, including improve transaction in TiKV and explore new transaction model.
+
+- Constitution: [中文](./constitution-zh_CN.md)
+- Membership: [English](./membership.md)
+- Workflow: [中文](./workflow-zh_CN.md)
+- Document Archive: 中文/English
+- Meeting Notes: English
+
+## Meetings
+
+* Regular SIG Meeting: 
+     * Bi-weekly on Wed 10am, Beijing Time (Tue 6pm PST) starting from 2020/02/12. 
```

Comment:
Is this correct?

Reply:
Will change it after you guys determined the meeting time.